core/predicate_check.go

```diff
@@ -31,12 +31,12 @@ func CheckPredicates(rules params.Rules, predicateContext *precompileconfig.Pred
 
 	predicateResults := make(map[common.Address][]byte)
 	// Short circuit early if there are no precompile predicates to verify
-	if len(rules.Predicates) == 0 {
+	if len(rules.Predicaters) == 0 {
 		return predicateResults, nil
 	}
 
 	// Prepare the predicate storage slots from the transaction's access list
-	predicateArguments := predicateutils.PreparePredicateStorageSlots(rules, tx.AccessList())
+	predicateArguments := predicate.PreparePredicateStorageSlots(rules, tx.AccessList())
```

Comment:
should we also remove "Predicate" from this fn name?

Reply:
you mean from `CheckPredicates`? I guess it's fine. Alternatively we can rename it to something like `RunPredicaters` etc.